### ISSUE TITLE
Code action generates invalid variable types for listener auth handlers

### ISSUE THREAD

**Issue by ldclakmal** [ballerina-platform/ballerina-lang#34489]
**Description:**
When creating variables with code actions for `authenticate` function of the listener auth handlers, the base module of the auth handler is referred to all the generated variable types instead of actual module.

Actual:
![image](https://user-images.githubusercontent.com/9639050/148022407-afea64af-ef35-4237-9667-5d2e8d61e424.png)

```ballerina
import ballerina/http;

http:ListenerFileUserStoreBasicAuthHandler handler = new;

service /foo on new http:Listener(9090) {
    resource function get bar(@http:Header { name: "Authorization" } string header) returns string|http:Unauthorized|http:Forbidden {
        http:UserDetails|http:Unauthorized authn = handler.authenticate(header);
        // ...
        return "Hello, World!";
    }
}
```

Expected:
![image](https://user-images.githubusercontent.com/9639050/148022424-f3f2f127-57f5-4183-b583-ec6385ea1aa2.png)

```ballerina
import ballerina/http;
import ballerina/auth;

http:ListenerFileUserStoreBasicAuthHandler handler = new;

service /foo on new http:Listener(9090) {
    resource function get bar(@http:Header { name: "Authorization" } string header) returns string|http:Unauthorized|http:Forbidden {
        auth:UserDetails|http:Unauthorized authn = handler.authenticate(header);
        // ...
        return "Hello, World!";
    }
}
```

The same issue is there for all of the Listener Auth Handlers of HTTP and gRPC.

| | Listener Auth Handler | Actual Variable Types | Expected Variable Types |
|---|---|---|---|
| 1. | `http:ListenerFileUserStoreBasicAuthHandler` | `http:UserDetails\|http:Unauthorized` | `auth:UserDetails\|http:Unauthorized` |
| 2. | `http:ListenerLdapUserStoreBasicAuthHandler` | `http:UserDetails\|http:Unauthorized` | `auth:UserDetails\|http:Unauthorized` |
| 3. | `http:ListenerJwtAuthHandler` | `http:Payload\|http:Unauthorized` | `jwt:Payload\|http:Unauthorized` |
| 4. | `http:ListenerOAuth2Handler` | `http:IntrospectionResponse\|http:Unauthorized\|http:Forbidden` | `oauth2:IntrospectionResponse\|http:Unauthorized\|http:Forbidden` |

**Affected Versions:**
Ballerina 2201.0.0 (Swan Lake) Snapshot Build 1

**OS, DB, other environment details and versions:**
MacOS Monterey

**Comment by dulajdilshan** [comment]
@ldclakmal I think you still has point the ballerina version to version beta 6.

**Comment by ldclakmal** [comment]
Yes. It is fixed in Ballerina 2201.0.0 (Swan Lake) Snapshot Build 1 pack. Hence closing the issue.
